extensions/amp-subscriptions/0.1/test/test-service-adapter.js

```diff
@@ -52,6 +52,14 @@ env => {
     serviceAdapter = new ServiceAdapter(subscriptionService);
   });
 
+  describe('getEncryptedDocumentKey', () => {
+    it('should call getEncryptedDocumentKey of subscription service', () => {
+      const stub = sandbox.stub(subscriptionService, 'getEncryptedDocumentKey');
+      serviceAdapter.getEncryptedDocumentKey('serviceId');
+      expect(stub).to.be.calledOnce;
```

Comment:
If we do go down this path, we should also check `calledWith('serviceId')` and that it returns the same value as the service returns. Otherwise, it's just a basic call propagation and doesn't have to be tested.

Reply:
done